README.md

```diff
@@ -584,6 +584,10 @@ Explore various applications that utilize the Tatum SDK. These examples help ill
 - [Metamask portfolio](https://github.com/tatumio/example-apps/tree/master/Metamask/portfolio)
 - [Metamask transfer](https://github.com/tatumio/example-apps/tree/master/Metamask/transfer)
 
+## Extension Ecosystem
+
+Learn more about Tatum SDK Extension ecosystem here - [Tatum SDK Extensions](https://github.com/tatumio/ecosystem-addons)
```

Comment:
I would mention here the example of using the extensions, even tough we have a repo, but that one is still empty :-D

Reply:
Added example usage of "any Tatum SDK Extension"... I'll be raising PR to the other repo shortly but that extension depends on this PR being in :)